src/commands/deploy/deploy.ts

```diff
@@ -88,6 +95,26 @@ async function deploy(actionContext: IActionContext, arg1: vscode.Uri | string |
     await notifyDeployComplete(context, node, context.workspaceFolder.uri.fsPath);
 }
 
+async function updateWorkerProcessTo64BitIfRequired(context: IDeployContext, siteConfig: WebSiteManagementModels.SiteConfigResource, node: SlotTreeItemBase, language: ProjectLanguage): Promise<void> {
+    const functionProject: string | undefined = await tryGetFunctionProjectRoot(context.workspaceFolder.uri.fsPath);
+    if (functionProject === undefined) {
+        throw new Error(localize('failedToFindFuncHost', 'Unable not locate Azure Functions project.'));
+    }
+    const projectFiles: string[] = await dotnetUtils.getProjFiles(language, functionProject);
+    if (projectFiles.length === 0) {
+        throw new Error(localize('unableToFindProj', 'Unable to detect project file.'));
+    }
+    const mainProject: string = path.join(functionProject, projectFiles[0]);
+    const platformTarget: string | undefined = await dotnetUtils.getPlatformTarget(mainProject);
+    if (platformTarget === 'x64' && siteConfig.use32BitWorkerProcess === true) {
+        const config: WebSiteManagementModels.SiteConfigResource = {
+            use32BitWorkerProcess: false
+        };
+        context.stopAppBeforeDeploy = true;
```

Comment:
Did you add this because you have to restart a function app every time you update the bitness? If so, that's not quite how this property was meant to be used. If a restart is necessary, I'd rather just do the full restart here, aka before the deploy

Reply:
if we don't restart the app, then we get an error `Function host is not running`

Reply:
In that case, I would before if you called `restartFunctionApp` right after you update the site config, instead of setting `context.stopAppBeforeDeploy`

Reply:
I tested few times by manually switching app to 32 bit and making local project 64 bit. In all cases, function app had no issues. So, I am removing the code to restart app.

Reply:
I guess this fix is not live yet on https://ms.portal.azure.com/ ? I'm still seeing this problem ...

Reply:
Also, how does one change a function app to 64bit using an "az command"  - for example I want to create my "az functionapp create" my app as 64bit to start with...